### PR TITLE
Minor edits to preface and chapter 1

### PR DESCRIPTION
**Pull request by tgerke**

### CODE REVIEW

**Review by malcolmbarrett — APPROVED** [review]

**Comment by malcolmbarrett** [comment]
thanks!

File: chapters/chapter-01.qmd [hunk 1]
```diff
@@ -519,7 +519,7 @@ ggplot(data, aes(x = x, y = y)) +
 Asking good causal questions means we map the *question* to the observable *evidence*.
 Let's return to the smoking example.
 Our initial question was: *Does smoking causes lung cancer?*; using our study, the evidence shows *For people who smoke 15+ cigarettes a day, reducing smoking by 50% reduces the risk of lung cancer over 5-10 years*.
-Do these match?
+Do the answer match the question?
```
Comment (LucyMcGowan) [inline chapters/chapter-01.qmd:522]:
```suggestion
Does the answer match the question?
```

**Review by LucyMcGowan — APPROVED** [review]
looks great!

File: chapters/chapter-01.qmd (hunk 1 above)
Comment (malcolmbarrett) [reply chapters/chapter-01.qmd:522]: I already merged but I'll address this

**Pull request merged**